scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -81,11 +81,9 @@
           {% if config.SQLALCHEMY_DATABASE_URI %}
             <div class="d-flex flex-wrap">
               {% for gene in variant.genes %}
-                <span class="ms-5">
-                  <a class="btn btn-sm btn-secondary" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
-                    Gene coverage:{{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
-                  </a>
-                </span>
+                <a class="btn btn-sm btn-secondary" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
```

Comment:
```suggestion
                <a class="btn btn-sm btn-secondary text-white" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
```

Reply:
Yes it doesn't work, but why the other links like igv and alamut do? Weird!